### PR TITLE
Release/gfsda.v16

### PR DESCRIPTION
**Pull request by ADCollard**
This PR fixes a typo in the exglobal_atmos_analysis.sh script.  NOAA-21 ATMS and CrIS were pointing to a temporary bufr files that is not present now that the data are operationally dumped.

- [x] Bug fix (non-breaking change which fixes an issue)

**How Has This Been Tested?**

The revised script as been tested in a single cycle and the results are as expected.
  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] New and existing tests pass with my changes
- [x] Any dependent changes have been merged and published

### CODE REVIEW

**Review by RussTreadon-NOAA — APPROVED** [review]
Change is straightforward.   The `OBS_INPUT` namelist referenced local files `atmbufr21` and `crisbufr21`.  The script copied the ATMS and CrIS bufr dump files to local files `atmsbufr` and `crisbufr`.   This PR removes the suffix `21` from local filenames in the `OBS_INPUT` namelist.

Approved pending peer reviews & approvals.

**Review by HaixiaLiu-NOAA — APPROVED** [review]
changes are good to go.

**Review by emilyhcliu — APPROVED** [review]
@ADCollard Good to spot the bug.  You  must receive separate BUFR files containing only N21 data for testing.  
Changes look good.  Approved!

**Comment by RussTreadon-NOAA** [comment]
Recreate tag [gfsda.v16.3.10](https://github.com/NOAA-EMC/GSI/releases/tag/gfsda.v16.3.10).  Done at [7fc1a3d](https://github.com/NOAA-EMC/GSI/commit/7fc1a3d2a01bb5730accf3bf4dc3367c2a0bba5e).

**Comment by ADCollard** [comment]
@emilyhcliu @HaixiaLiu-NOAA @RussTreadon-NOAA Thanks!

**Pull request merged**
